contents/blog/product-engineer-vs-product-manager.md

```diff
@@ -0,0 +1,75 @@
+---
+title: "Product engineer vs product manager: what's the difference"
+date: 2023-01-20
+author: ["ian-vanagas"]
+showTitle: true
+rootpage: /blog
+sidebar: Blog
+hideAnchor: true
+featuredImage: ../images/blog/posthog-engineering-blog.png
+featuredImageType: full
+category: Engineering
+tags: 
+    - Explainers
+---
+
+A product engineer is a combination of a software engineer and a product manager. We already compared [product engineers and software engineers](/blog/product-engineer-vs-software-engineer), and now is the time for product managers.
+
+Product engineers and product managers want to build a great product but differ in how they accomplish this. The product engineer ships code while the product manager researches, plans, and manages development. To understand both roles more, we:
+- compare how both work to build a product
+- explain how the rise of product engineers impacts product managers
+- explore how they can work together
+
+## How do product engineers and product managers build a product?
+
+Developing a product is a journey. The goal of the journey is to launch a successful product or feature that solves user problems. Both product engineers and managers share this goal. What is different is the path they take, which we walk through here.
+
+### Beginning the product development journey
+
+Both product engineers and managers are responsible for the start of the journey, figuring out what to work on. This means talking to users, analyzing usage data, understanding the competitive landscape, and using all this knowledge to prioritize work. Both translate user needs into product requirements.
+
+Product managers go into further depth here, often writing up reports and recommendations on their findings. This requires stronger analysis and research skills. Product engineers write issues and todos then move onto building as fast as possible.
```

Comment:
```suggestion
Product managers go into further depth here, often writing up reports and recommendations on their findings. This requires stronger analysis and research skills. Product engineers write issues and todos then move onto building as fast as possible. This makes product engineers ideal for early-stage startups, where shipping quickly and iterating is preferable to drawn out discovery work.
```

Feel free to take or leave this suggestion, but felt it's worth reinforcing this point when it's relevant.